src/ga.js

```diff
@@ -0,0 +1,9 @@
+export const googleAnalytics = (gaID) => {
+  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+  })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
+
+  ga('create', gaID, 'auto');
+  ga('send', 'pageview');
+};
```

Comment:
This was hard to read. Just wondering why you don't want to prettier-fy this?

Reply:
This is basically just a pre-minified extract from Google, formatting the chunk wouldn't really improve it substantially and makes it a little less recognizable: that bit of code is actually very familiar to many people who have worked with google analytics previously.